examples/juniper/cptx-ixia/cptx-ixia.pb.txt

```diff
@@ -0,0 +1,110 @@
+name: "ixia-c"
+nodes: {
+    name: "otg"
+    vendor: KEYSIGHT
+    version: "0.0.1-3423-local"
```

Comment:
Please use the special `version: "0.0.1-9999"`

Reply:
Done